pkg/ocicni/ocicni.go

```diff
@@ -334,11 +334,9 @@ func (plugin *cniNetworkPlugin) syncNetworkConfig() error {
 	}
 
 	plugin.Lock()
-	defer plugin.Unlock()
-	if plugin.defaultNetName == "" {
```

Comment:
doesn't removing this mean default is never set?

Reply:
It's now always set, which was my main intention :point_down: